packages/itwin/grouping-mapping-widget/src/components/GroupingMappingContext.tsx

```diff
@@ -48,6 +50,14 @@ export interface GroupingMappingContextProps {
    * A custom implementation of MappingClient.
    */
   client?: IMappingsClient;
```

Comment:
I think this should be renamed to mappingsClient.

Reply:
Changed to mappingsClient.